### PR TITLE
create `/rereview` command

### LINKED ISSUE #40: /rereview
**Describe the feature**
A command to notify the revisors that the changes was made

<details open>
  <summary>
    <b>Description</b>
  </summary>
 A command to notify the revisors that the changes was made
</details>

<br>

<details> 
  <summary>
    <b>Use Case</b>
  </summary>
  
when the pr owner do the requested changes on pr and wants to notify the revisors about that he calls this command
</details>

### PR DESCRIPTION
**Pull request by hxsggsz**
Closes #40 

<details open> 
  <summary>
    <b>Feature</b>
  </summary>

creates the `/rereview` command to people notify when their's PR are ready to review again

</details>

<details open> 
  <summary>
    <b>Changelog</b>
  </summary>
create folder rereview and a file rereview inside command folder
</details>

<details open> 
  <summary>
    <b>Visual evidences :framed_picture:</b>
  </summary>
  
image

</details>

<details open> 
  <summary>
    <b>Checklist</b>
  </summary>

- [x] Issue linked
- [x] Build working correctly
- [x] Tests created
</details>

<details open> 
  <summary>
    <b>Additional info</b>
  </summary>
N/A
</details>

### CODE REVIEW

File: src/commands/changes/changes.ts [hunk 1]
```diff
@@ -3,7 +3,9 @@ import { CommandInteraction, SlashCommandBuilder, Client } from 'discord.js'
 
 export const data = new SlashCommandBuilder()
   .setName('changes')
-  .setDescription('Avisar sobre mudanças no pr')
+  .setDescription(
+    'Avisar sobre possíveis mudanças necessárias a serem feitas no pr'
```
Comment (k1nha) [inline src/commands/changes/changes.ts:7]:
```suggestion
    'Avise sobre possíveis mudanças necessárias a serem feitas no pr'
```

File: src/commands/rereview/rereview.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import { CommandInteraction, SlashCommandBuilder, Client } from 'discord.js'
+
+export const data = new SlashCommandBuilder()
+  .setName('rereview')
+  .setDescription(
+    'Avisar aos revisores que seu PR está pronto para ser revisado novamente'
+  )
+
+export async function execute(interaction: CommandInteraction, client: Client) {
+  const channel = client.channels.cache.get(interaction.channelId)
```
Comment (k1nha) [inline src/commands/rereview/rereview.ts:11]: precisa de tratamento? caso não ache

**Review by k1nha — APPROVED** [review]

File: src/commands/pulls/pulls.ts [hunk 3]
```diff
@@ -84,7 +84,7 @@ export async function execute(interaction: CommandInteraction, client: Client) {
       .setCustomId(`select-user`)
       .setPlaceholder('Pessoas para revisar')
       .setMinValues(1)
-      .setMaxValues(10)
+      .setMaxValues(2)
```
Comment (aalmeida00) [inline src/commands/pulls/pulls.ts:87]: No máximo 2 pq? É na real no minimo 2 e no maximo N (dá pra deixar tipo 5-6)

File: src/utils/bot/prContent.spec.ts [hunk 4]
```diff
@@ -50,14 +51,14 @@ describe('prContent', () => {
...
-                '**Repositório:** repository test\n **Pull Request ID:** not a number',
+                '**Revisores:** mock user \n**Repositório:** repository test\n **Pull Request ID:** not a number',
             }),
           }),
         },
       })
 
       const mockResult = await prContent(mockChannel)
-
+      console.log(mockResult)
```
Comment (aalmeida00) [inline src/utils/bot/prContent.spec.ts:61]: Pq o console log aqui"?

**Review by aalmeida00 — DISMISSED** [review]
Algumas dúvidas

File: src/commands/pulls/pulls.ts (hunk 3 above)
Comment (hxsggsz) [reply src/commands/pulls/pulls.ts:87]: eu coloquei no máximo 2 pq agora só precisa de dois reviews pra aprovar o pr

File: src/commands/rereview/rereview.ts (hunk 2 above)
Comment (hxsggsz) [reply src/commands/rereview/rereview.ts:11]: percebi que a variavel channel pode ser undefined tambem, coloquei um ifzinho humilde la

**Review by k1nha — APPROVED** [review]

**Pull request merged**